src/main/java/com/develhope/spring/deals/models/Order.java

```diff
@@ -1,30 +1,41 @@
 package com.develhope.spring.deals.models;
 
-import com.develhope.spring.users.models.User;
 import com.develhope.spring.vehicles.models.Vehicle;
+import jakarta.persistence.*;
 
 /**
  * Represents an order or purchase. This class includes details about
  * the payment status, order status, and identifies the vehicle associated with the order.
  */
-
+@Entity
+@Table(name = "orders")
 public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
     private boolean downPayment;
+
+    @Column(nullable = false)
     private OrderStatus orderStatus;
+
+    @Column(nullable = false)
     private boolean isPaid;
+    @OneToOne
     private Vehicle vehicle;
 
```

Comment:
Mi sembra se ne stesse occupando già Claudio di questa cosa all'interno della sua pull request di Create Order. Comunque dovresti aggiungere anche allora un riferimento a User user oltre che Vehicle vehicle;

Reply:
Ho visto che la pr di @Calyen93 è stata taggata come obsoleta, ho così pensato di portare avanti questa comunque

Reply:
è preferibile ManyToOne in quanto lo stesso veicolo può apparire in diversi ordini